src/config.js

```diff
@@ -0,0 +1,38 @@
+import creditCardType, { types as cardTypes } from 'credit-card-type';
```

Comment:
You can merge this code with the packages abstractions

Reply:
done, all helpers are now combined in the `utils/cardHelpers.js` file